lib/discourse_theme/cli.rb

```diff
@@ -167,6 +183,94 @@ def run(args)
         else
           UI.info "Manage: #{client.root}/admin/customize/themes/#{theme_id}"
         end
+      elsif command == "rspec"
+        index = dir.index("/spec")
+        spec_path = "/spec"
+
+        if index
+          spec_path = dir[index..-1]
+          dir = dir[0..index - 1]
+        end
+
+        raise DiscourseTheme::ThemeError.new "'#{dir} does not exist" unless Dir.exist?(dir)
+
+        # Checks if the container is running
+        container_name = DISCOURSE_TEST_DOCKER_CONTAINER_NAME
+        is_running = false
+        container_exists = false
+
+        if !(
+             output =
+               execute(command: "docker ps -a --format '{{json .}}' | grep #{container_name}")
+           ).empty?
+          container_exists = true
+          is_running = JSON.parse(output)["State"] == "running"
+        end
+
+        basename = Pathname.new(dir).basename.to_s
+        verbose = !!args.delete("--verbose")
+        headless = !!args.delete("--headless")
+
+        if !is_running || args.delete("--rebuild")
```

Comment:
What if a container is already running, but it has a volume mount to a different theme? Can we detect that and stop/start it?

Reply:
This should be resolved in https://github.com/discourse/discourse_theme/pull/28/commits/2a0c8d9fc7f7e30cd6da14ef1365575ad20c9319 since the volume mount is no longer tied directly to a theme.